script/bootstrap_server

```diff
@@ -6,21 +6,13 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-echo "Installing dependencies..."
-# Requirements_all.txt states minimum pip version as 7.0.0 however,
-# parameter --only-binary doesn't work with pip < 7.0.0. Causing
-# python3 -m pip install -r requirements_all.txt to fail unless pip upgraded.
-
-if ! python3 -c 'import pkg_resources ; pkg_resources.require(["pip>=7.0.0"])' 2>/dev/null ; then
-  echo "Upgrading pip..."
-  python3 -m pip install -U pip
-fi
-python3 -m pip install -r requirements_all.txt
+echo "Installing Home Assistant in development mode..."
+python3 setup.py develop
```

Comment:
Ah yes, you're Bootstrap should not do that. right. https://github.com/github/scripts-to-rule-them-all#scriptbootstrap